boards/qemu_i486_q35/layout.ld

```diff
@@ -22,6 +22,16 @@ MEMORY
  */
 PAGE_SIZE = 4K;
 
+/**
+ * The order matters here as we need to make sure that qemu finds the multiboot header within the
+ * first 8K of the ELF. The multiboot header is placed at the begining of the .text section, so
+ * this section has to be is placed first in the file.
+ *  
+ * This is the reason why this INCLUDE has to be before the .pages declaration
+ */
```

Comment:
```suggestion
/**
 * The order matters here as we need to make sure that qemu finds the multiboot header within the
 * first 8K of the ELF. The multiboot header is placed at the beginning of the .text section, so
 * this section has to be is placed first in the file.
 *  
 * This is the reason why this INCLUDE has to be before the .pages declaration
 */
```

Reply:
Also, what actually _is_ the multiboot header? If I wanted to point someone to it, where would I send them?

Reply:
I added a link and a small description in both linker scripts, `kernel_layout.ld` and the board's `layout.ld`.